scripts/install_apt_deps.sh

```diff
@@ -1,21 +1,92 @@
+pkg_warn_msg_realsense="Currently, IntelRealSense does not support binary package installations for Ubuntu 20 LTS kernels, or other non-Ubuntu Linux distributions. If your project requires IntelRealSense support, please build and install the IntelRealSense SDK from source. For more information, visit '${pkg_install_url_realsense}' and '${pkg_build_url_realsense}'."
+
+# Check for distribution support of IntelRealSense
+# For supported distributions, automatically add the required install
+# package dependencies
+echo "Detected OS: '${PRETTY_NAME}'"
+if [ "${ID}" == "ubuntu" ] && [ "${VERSION_ID}" == "18.04" ]; then
+    use_realsense="yes"
+    pkg_dep_list+="${pkg_dep_list_realsense} "
+elif [ "${ID}" == "ubuntu" ] && [ "${VERSION_ID}" == "20.04" ]; then
+    use_realsense="no"
+    print_warning "${pkg_warn_msg_realsense}"
+else
+    use_realsense="no"
+    print_warning "${pkg_warn_msg_realsense}"
+fi
+
+
+### Package dependencies installation ###
+
 sudo apt install -y software-properties-common curl gnupg2
 curl https://apt.kitware.com/keys/kitware-archive-latest.asc | sudo apt-key add -
 sudo apt-add-repository -u -y "deb https://apt.kitware.com/ubuntu/ ${UBUNTU_CODENAME} main"
-sudo apt-key adv --keyserver keys.gnupg.net --recv-key F6E65AC044F831AC80A06380C8B3A55A6F3EFCDE || sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key F6E65AC044F831AC80A06380C8B3A55A6F3EFCDE
-sudo add-apt-repository "deb http://realsense-hw-public.s3.amazonaws.com/Debian/apt-repo bionic main" -u
+
+# If supported, add the gpg keys and repository for IntelRealSense
+if [ "${use_realsense}" == "yes" ]; then
+    sudo apt-key adv --keyserver keys.gnupg.net --recv-key F6E65AC044F831AC80A06380C8B3A55A6F3EFCDE || sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key F6E65AC044F831AC80A06380C8B3A55A6F3EFCDE
+    sudo add-apt-repository "deb http://realsense-hw-public.s3.amazonaws.com/Debian/apt-repo bionic main" -u
+fi
+
 sudo add-apt-repository -u -y ppa:graphics-drivers/ppa
 sudo add-apt-repository -u -y ppa:deadsnakes/ppa
-sudo apt-get install -y \
-	 git clang-10 make cmake libc++-dev libc++abi-dev unzip \
-	 libsqlite3-dev libeigen3-dev libboost-all-dev libatlas-base-dev libsuitesparse-dev libblas-dev libtbb-dev \
-	 glslang-tools libsdl2-dev libglu1-mesa-dev mesa-common-dev freeglut3-dev libglew-dev glew-utils libglfw3-dev \
-	 libusb-dev libusb-1.0 libudev-dev libv4l-dev libhidapi-dev \
-	 build-essential libx11-xcb-dev libxcb-glx0-dev libxcb-randr0-dev libxrandr-dev libxkbcommon-dev libwayland-dev \
-	 libepoxy-dev libdrm-dev libgbm-dev libx11-dev libvirglrenderer-dev libpulse-dev \
-	 libgtest-dev pkg-config libgtk2.0-dev wget xvfb librealsense2-dkms librealsense2-utils \
-     libjpeg-dev libpng-dev libtiff-dev libvtk6-dev libgtk-3-dev libparmetis-dev gfortran \
-	 libgflags-dev libgoogle-glog-dev \
+sudo apt-get install -y ${pkg_dep_list}
```

Comment:
Let's just move to `apt` instead of `apt-get`. We use `apt` on line 80.

Reply:
I thought `apt` wasn't safe to use within scripts, because the interface is not standardized.

```
SCRIPT USAGE
       The apt(8) commandline is designed as a end-user tool and it may change
       the output between versions. While it tries to not break backward
       compatibility there is no guarantee for it either. All features of
       apt(8) are available in apt-cache(8) and apt-get(8) via APT options.
       Please prefer using these commands in your scripts.
```

Reply:
Sounds good.